pkg/rke/k8s_rke_system_images.go

```diff
@@ -9342,6 +9342,51 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.20",
 		},
+
+		// Enabled in Rancher v2.8.x
+		"v1.27.5-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.7",
+			Kubernetes:                "rancher/hyperkube:v1.27.5-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.89",
```

Comment:
We need to bump rke-tools to v0.1.90 for https://github.com/rancher/rancher/issues/42103

Reply:
Ah! just a sec. Updating

Reply:
@kinarashah Updated the PR